tools/mo/openvino/tools/mo/front/extractor.py

```diff
@@ -1092,10 +1092,12 @@ def add_input_ops(graph: Graph, user_defined_inputs: dict, before_infer: bool):
                         op_name = smart_node.soft_get('name')
                         if graph.has_tensor_name(op_name):
                             continue
-                        fw_info = []
-                        if 'fw_tensor_debug_info' in smart_node.out_edge(0):
-                            fw_info += smart_node.out_edge(0)['fw_tensor_debug_info']
-                        smart_node.out_edge(0)['fw_tensor_debug_info'] = fw_info + [(op_name, op_name)]
+                        out_edges = list(graph.out_edges(op_name, data=True))
```

Comment:
Is it covered by any test?

Reply:
Tensor names functionality is tested in mo_args_tests on small set of models. To cover this particular case we need to add original model to tests share and test for it in mo_args_tests. It can be done in a separate PR.

Reply:
Please fill ticket for test adding to remember it.

Reply:
Created ticket: CVS 82195